api/audit/signals.py

```diff
@@ -61,6 +61,7 @@ def signal_wrapper(sender, instance, **kwargs):
             RelatedObjectType.FEATURE.name,
             RelatedObjectType.FEATURE_STATE.name,
             RelatedObjectType.SEGMENT.name,
+            RelatedObjectType.EF_VERSION.name,
```

Comment:
This is essentially the only thing that needed to change to fix the issue. The rest of the changes are related to the tests, improving the integration for EF versions, or refactoring.